src/index.tsx

```diff
@@ -28,6 +28,9 @@ console.log('HyperSDKReactVersion', HyperSdkReact);
 
 if (Platform.OS === 'android') {
   HyperSdkReact.updateBaseViewController = () => {};
+  HyperSdkReact.updateMerchantViewHeight = () => {
```

Comment:
can we use the same signature here as we have for iOS? 

```suggestion
  HyperSdkReact.updateMerchantViewHeight = (tag: string, height: number) => {
```

Reply:
this change is still pending